Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -86,6 +86,8 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder
 
 - (void)commonInit
 {
+    self.groupId = [Randomness generateRandomBytes:16];
```

Comment:
To ensure that the "new group" view accurately reflects the new group's conversation color, we need to eagerly generate the group id.